src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
@geekyd I always suggest measuring the improvements. 
Wherever a JS is involved making a CSS change, most importantly `Layouting` which is changing here, the whole cycle of Layout => Paint => Composite takes place which affects the performance.
Also if it contributes to performance increment of say 1ms ( which is very small ) but it will vary linearly with the data.   
Please carry the discussions on the issue.